client/components/payment-method-unavailable-due-conflict-pill/index.js

```diff
@@ -48,25 +45,22 @@ const IconComponent = ( { children, ...props } ) => (
 	</IconWrapper>
 );
 
-const PaymentMethodUnavailableDueConflictPill = ( { id, label } ) => {
+const PaymentMethodRequiredForOCPill = ( { id, label } ) => {
```

Comment:
I think the changes in this file were intended for the new file `client/components/payment-method-required-for-oc-pill/index.js`